### DISCUSSION TITLE
Question on URI format for OTLP exporter

### DISCUSSION THREAD

**Discussion by davendu** [open-telemetry/opentelemetry-java#3278] [Q&A] [answered]
Hi. I noticed in https://github.com/open-telemetry/opentelemetry-java/blob/50408d499f85d5761d0a5ed9bf9d77d5ff01fff5/exporters/otlp/trace/src/main/java/io/opentelemetry/exporter/otlp/trace/OtlpGrpcSpanExporterBuilder.java#L87-L89

The builder only accepts URI starts with "http" or "https". But later, the exporter is built with gRPC, which is neither HTTP or HTTPS. So what's the situation here, is it reporting to collector in HTTP(s) or gRPC?

**Comment by jkwatson** [comment, marked as answer]
grpc is transported via the HTTP protocol to the collector, so this is appropriate.

**Reply by davendu** [reply]
thanks!
